CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Multiple solutions are allowed for a single symbolic expression
 - Aliasing works much better for symbolic and concrete addresses
 - Constant propagation for symbolic values
+- One more simplification rule for `ReadByte` when the `CopySlice` after it
```

Comment:
Please mention also the added simplification for `ReadWord` in the entry.

Reply:
Oh, good catch. Fixed!